### ISSUE TITLE
HTTP tunnel 'command_line' specifies invalid scheme ''. use 'https' or 'http' after newest update using static domain

### ISSUE THREAD

**Issue by cobrani** [ngrok/ngrok#59]
Since the latest update to version  3.18.3, I get the error

`ERROR:  HTTP tunnel 'command_line' specifies invalid scheme ''. use 'https' or 'http'` when using
`ngrok http --url=myurl.ngrok-free.app 5001`. I tried changing the order of the parameters, but still get the issue. Using an ephemeral domain via `ngrok http http://localhost:5001` works without any problems (macOS, Sonoma, 14.6.1).

**Comment by simonwhitaker** [comment]
If you replace `--url` with `--domain` it seems to do the trick.

**Comment by inconshreveable** [comment]
try:

```
ngrok http --url https://myurl.ngrok-free.app http://localhost:5001
```

I think it's not defaulting to an https URL properly as it should be and we'll get that and the error message fixed

**Comment by simonwhitaker** [comment]
@inconshreveable Thanks! Adding a scheme to the URL does the trick.

:x: Doesn't work:

```
ngrok http --url foo-bar-baz.ngrok-free.app 8000
```

✅ Does work:

```
ngrok http --url https://foo-bar-baz.ngrok-free.app 8000
```

**Comment by kaganjd** [comment]
hi all- requiring a scheme at the beginning of the URL was a regression in 3.18.3 that has now been fixed in 3.18.4- so something like `ngrok http --url=myurl.ngrok-free.app 5001` should work again and `ngrok http --url=https://myurl.ngrok-free.app 5001` will also work.

going to close this out but please feel free to reopen if you encounter this issue again on 3.18.4 or higher!
